File: leetcode_ly/java/String/ReverseString.java
public class Solution {
    public String reverseString(String s) {
        if(s.isEmpty()) return s;
        StringBuffer ss = new StringBuffer(s);
        int i = 0;
        int j = s.length() - 1;
        while(i < j){
            char temp = ss.charAt(i);
            ss.setCharAt(i,ss.charAt(j));
            ss.setCharAt(j,temp);
            i++;
            j--;
        }
        return ss.toString();
    }
}

File: leetcode_ly/c++/easy/ReverseString.cpp
class Solution {
public:
    string reverseString(string s) {
        int len = s.size();
        char a;
        for(int i = 0; i < len; i++){
            if(i < len - i - 1){
                a = s[i];
                s[i] = s[len - i - 1];
                s[len - i - 1] = a;
            }
        }
        return s;
    }
};